### PR TITLE
Sean discards

### PR DESCRIPTION
**Pull request by slucey**
Added discard data to catchIndex.

### CODE REVIEW

**Comment by slucey** [comment]
I didn't see the dev_branch when making this pull request.  Let me know if you want me to cancel and redo to the dev branch first.

**Review by sgaichas — APPROVED** [review]
I think this looks good @slucey, thanks!
Is this scaled to the GB EPU from the stat areas in lines 26-28 of pull_comland.R?
I think its ok to go into master instead of dev at this point

File: data-raw/R/create_fishery_index.R [hunk 1]
```diff
@@ -3,14 +3,28 @@
...
 #Sum over all catch variables for total catch
 spp <- as.numeric(unique(mskeyrun::focalSpecies$NESPP3))
-catchIndex <- comland$comland[NESPP3 %in% spp, .(value = sum(SPPLIVMT)),
+
+#Landings
+landIndex <- landings$comland[NESPP3 %in% spp, .(value = sum(SPPLIVMT)),
                               by = c('NESPP3', 'YEAR')]
-catchIndex[, variable := 'commerical landings']
-catchIndex[, units    := 'metric tons']
+landIndex[, variable := 'commerical landings']
```
Comment (sgaichas) [inline data-raw/R/create_fishery_index.R:17]: @slucey I just noticed that `commercial` has a typo here
(I didn't even see it but my code broke--I wasn't calling the variable before so didn't catch it)

**Review by sgaichas — CHANGES_REQUESTED** [review]
we probably should change this variable name

File: data-raw/R/create_fishery_index.R (hunk 1 above)
Comment (sgaichas) [reply data-raw/R/create_fishery_index.R]: I changed it in this file but dont have the source data to correct the dataset

Comment (slucey) [reply data-raw/R/create_fishery_index.R]: Thanks for "catching" that...I fixed the code and updated the data object.

**Comment by slucey** [comment]
> 

Yes...the `landings` and `discards` objects I pull in are created using the `pull_comlands.R` script which segregates out the GB catch.

**Review by sgaichas — APPROVED** [review]
thanks!

**Pull request merged**